### ISSUE TITLE
Review xref obsoleted at the source: DO 

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#4365]
This review is part of the analysis for the Mondo paper. 

The DO terms [here](https://docs.google.com/spreadsheets/d/1nGr_j-JevUelgQ8LusQ0Qp8LRP-E2RMF/edit?usp=sharing&ouid=115291133956454706378&rtpof=true&sd=true) are obsoleted in DO, but are still xref in Mondo. 

To do: review the Mondo terms with these DO xref, and either update to Mondo:equivalentObsolete, or obsolete the class in Mondo.

**Comment by nicolevasilevsky** [comment]
@sabrinatoro can you do a SPARQL query for the other xrefs that are on these terms? I think if they are live in other ontologies, we'll keep them in Mondo and just add Mondo:equivalentObsolete, as you mentioned.

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
I reviewed the terms. 
Most of the terms with the DO xref obsoleted were due to 
- obsoletion in DO possibly due to the fact that the term might not be a disease (we should review these at one point)
- merging in DO. I am not convinced that the corresponding terms should be merged in Mondo (they look like different concepts), but it wouldn't hurt to review these carefully too at one point)

I will create new issues for terms that require more urgent review.
